apps/modernization-api/src/main/resources/search/patient/patient.index.json

```diff
@@ -0,0 +1,632 @@
+{
+  "mappings": {
+    "properties": {
```

Comment:
we could put `"_source": {  "enabled": false },` here if we are certain we are not fetching documents

Reply:
awesome, let's add this when we re-evaluate the indexes.  The index will only be created if it doesn't already exist